example/index.js

```diff
@@ -0,0 +1,62 @@
+import { OnePasswordConnect, ItemBuilder } from "@1password/connect";
+import * as readline from "readline-sync"
+import steps from steps
```

Comment:
I'm getting an error when running the container. Was this accidentally left behind after removing a file?

Reply:
This line should definitely not be here. I'll investigate, to make sure there aren't any other remains of older versions of the example. Thanks for spotting this!

Reply:
This seems to be the only issue, removing this line seems to be fixing it, for me. Let me know if it's the same case for you. 😀